shell.nix

```diff
@@ -0,0 +1,20 @@
+# Update nixpkgs with:
+# nix-shell -p niv --run "niv update"
+
+let
+  sources = import ./nix/sources.nix;
+  pkgs = import sources.nixpkgs {};
+in
+pkgs.mkShell {
+  buildInputs = [
+    pkgs.bash
```

Comment:
This is where you choose your tools. Unlike dev-env, all of the binaries provided by each package is automatically added to PATH. On the one hand, this means you don't have to make the mapping explicitly and create symlinks manually; on the other hand, it means that you can't easily alias individual binaries in case of conflicts, and that your path gets "polluted" by all the binaries of each package.

Reply:
i guess same override applies if we want a specific version of the tool that does not align with the default pkgs.... version of that tool that is part of the `nixpkgs-unstable` version that we are using?

most specifically, we need particular versions of yq and nodejs/npm/yarn